save-backend/build.gradle.kts

```diff
@@ -6,6 +6,20 @@ import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 plugins {
     kotlin("jvm")
+    // this plugin will generate generateOpenApiDocs task
+    // running this task, it will writes the OpenAPI spec into a backend-api-docs.json file in save-backend dir.
+    id("org.springdoc.openapi-gradle-plugin") version "1.3.4"
+}
+
+openApi {
+    apiDocsUrl.set("http://localhost:5000/internal/v3/api-docs/latest")
+    outputFileName.set("$rootDir/save-backend/backend-api-docs.json")
```

Comment:
We could also add a validation task, that checks that current API isn't breaking compatibility with previous with the same version. But it can be done in the future

Reply:
Yep, later. I even do not think that versioning is was not so needed now